react/components/FloatingActionBar/README.md

```diff
@@ -0,0 +1,26 @@
+#### Floating Action Bar allow users to easily find the save button while editing an instance of an entity.
+
+### 👍 Dos
+
+- Only use when the action affects the whole page.
+- The actions on this component may be used to help user cancel or proceed a new change.
+- This component can be used anywhere but it will always show at the bottom og the page.
+
+### 👎 Don'ts
+
+- Avoid to use this component for specific contexts.
+- The component won't be afected by the page overflow.
+
+### Related components
+
+- There is no related components, **yet**.
+
+### Usage
+
+```js
+<FloatingActionBar
+    onSave={() => save(updated)}
```

Comment:
How does it work if there is no `save` function defined?

Reply:
It will throw an error, I'm gonna change it to an alert anyway, more descriptive I think